CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -43,7 +43,7 @@ <h2 class="definition-title definition-title--search-result">
             {% if result.is_inflection %}
                 <p class="reference-to-lemma" data-cy="reference-to-lemma">
                     Form of
-                    <a class="reference-to-lemma__lemma" href="{% url 'cree-dictionary-index-with-lemma' result.lemma_wordform.id %}"
+                    <a class="reference-to-lemma__lemma" href="{{ result.lemma_url }}"
```

Comment:
Even with my proposed changes, using `result.lemma_url` should remain the same. 👍